### PR TITLE
This fixed the resizing issue on mobile for the calendar

### PR DESCRIPTION
**Pull request by henlegay**

### CODE REVIEW

**Comment by sebastian-king** [comment]
This does indeed work! Nice job deploying it to dev. I think there are some things we want to discuss about this task in general, such as if we want to stick with the Google calendar classic view or move to something more modern (I think we previously talked about finding a better calendar library?). We can talk about these and review this PR together in our regularly scheduled meetings that @Peyton232 needs to schedule!

**Comment by ntindle** [comment]
Let’s merge this since it works now and talk about upgrading the calendar later?

**Comment by sebastian-king** [comment]
Good point! Once the .idea files have been removed I’m happy to approve the PR

**Comment by ntindle** [comment]
You've done it!

File: events.php [hunk 1]
```diff
@@ -2,21 +2,28 @@
 require('template/top.php');
 head('Events Calendar', true);
 ?>
+    <style>
```
Comment (ntindle) [inline events.php:5]: @sebastian-king are we putting stuff in global CSS?

**Review by ntindle — APPROVED** [review]

**Comment by Peyton232** [comment]
looks good

**Review by sebastian-king — CHANGES_REQUESTED** [review]
Can we re-add *.log to the .gitignore file?

**Review by ntindle — DISMISSED** [review]

**Comment by ntindle** [comment]
LGTM (looks good to me)

**Comment by Peyton232** [comment]
Shouldn't we make it *.idea

**Comment by ntindle** [comment]
.idea removes the folder 

*.idea removes anything with that extension

**Comment by ntindle** [comment]
.idea/
And 

*.idea could be prudent

**Comment by ntindle** [comment]
Just copy and paste this to the end of the file and let's call it a day

https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore

**Review by sebastian-king — DISMISSED** [review]

**Review by ntindle — APPROVED** [review]
LGTM for real this time

**Comment by sebastian-king** [comment]
Now that the PR is merged can you update the Jira ticket status to Ready for Production?

**Pull request merged**